services/src/pro/apidoc.rs

```diff
@@ -408,28 +408,24 @@ impl Modify for ApiDocInfo {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::contexts::Session;
-    use crate::pro::contexts::ProInMemoryContext;
-    use crate::pro::users::UserAuth;
     use crate::pro::util::tests::send_pro_test_request;
-    use geoengine_datatypes::util::test::TestDefault;
+    use crate::pro::util::tests::with_pro_temp_context;
 
     #[test]
     fn can_resolve_api() {
-        crate::api::can_resolve_api(ApiDoc::openapi());
+        crate::util::openapi_examples::can_resolve_api(ApiDoc::openapi());
     }
 
-    #[tokio::test]
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
```

Comment:
didn't you want to create a macro that captures test, tokio and the pg context?

Reply:
I thought about it, but I wasn't really sure about how to do it, because the #[toko::test] is also a macro, so we'd have nested macros, and also: tests can't have arguments, so I wasn't really sure how to properly implement this.

That's why I stuck with the current approach of multi_thread flavor and the with_temp_context methods.

Reply:
* because the #[toko::test] is also a macro
   *  -> in the worst case, we just use the code that is created by the macro. it is just some lines.
 * tests can't have arguments
   * -> our tests could. we would just put our test method with arguments inside a test method

Reply:
I think it's possible to do this, but I would vote for doing this in a separate task.

Reply:
Yeah, let's get this merged first.